src/models/income.models.ts

```diff
@@ -26,8 +26,12 @@ export interface IncomesWithImportDate extends IncomesWithYear {
   importDate: string | null;
 }
 export interface IncomesDb extends IncomesWithImportDate {
-  id: string | null;
+  id: string | number;
 }
```

Comment:
First - if we have unused interface (or another code lines), we can delete it.  
Second - Do we use id in number type?